src/commands/RPG/RPG.ts

```diff
@@ -207,23 +233,39 @@ export class RPG {
       summary: summary,
       challenger: challenger,
       accepter: accepter,
+      messageId: messageId,
+      creationTime: Date.now(),
     }
 
     return result
   }
 
   @Slash('character', { description: 'Show off your character sheet' })
-  async character(interaction: CommandInteraction) {
+  async character(
+    @SlashOption('silent', { type: 'BOOLEAN' })
+    @SlashOption('name', { type: 'STRING', required: false })
+    silent: boolean,
+    name: string,
+    interaction: CommandInteraction
+  ) {
     const callerMember = getCallerFromCommand(interaction)
     const callingUser = callerMember?.user
-
     if (callingUser) {
-      const userDBRecord = await this.getUserFromDB(callerMember.user.id)
-      const eloBandIcon = this.getBandForEloRank(userDBRecord.eloRank)
-      const character = new Character(callingUser, callerMember.nickname ?? undefined)
-      interaction.reply({ embeds: [character.toEmbed(eloBandIcon.icon)] })
+      if (name) {
+        const character = new Character(callingUser, name)
+        interaction.reply({ embeds: [character.toEmbed('')], ephemeral: silent })
+      } else {
+        const userDBRecord = await this.getUserFromDB(callerMember.user.id)
+        const eloBandIcon = this.getBandForEloRank(userDBRecord.eloRank)
+        const character = new Character(
+          callingUser,
+          callerMember.nickname ?? callerMember.user.username,
+          callerMember.nickname ?? undefined
+        )
+        interaction.reply({ embeds: [character.toEmbed(eloBandIcon.icon)], ephemeral: silent })
+      }
     } else {
-      interaction.reply('Username undefined')
+      interaction.reply({ content: 'Username undefined', ephemeral: silent })
```

Comment:
We usually try and make errors ephemeral, while this isn't *really* an error I think it makes sense for it to always be ephemeral.